types/helpers.d.ts

```diff
@@ -1,69 +1,100 @@
 import Vue from 'vue';
 import { Dispatch, Commit } from './index';
 
-type Dictionary<T> = { [key: string]: T };
-type Computed = () => any;
-type MutationMethod = (...args: any[]) => void;
-type ActionMethod = (...args: any[]) => Promise<any>;
-type CustomVue = Vue & Dictionary<any>
-
-interface Mapper<R> {
-  (map: string[]): Dictionary<R>;
-  (map: Dictionary<string>): Dictionary<R>;
+/**
+ * Utility types to declare helper types
+ */
+type Computed<R> = () => R;
+type Method<R> = (...args: any[]) => R;
+type MutationMethod<P> = (payload: P) => void;
+type ActionMethod<P> = (payload: P) => Promise<any>;
```

Comment:
I'm fine with untyped action return type. In fact, users can register multiple actions so the return type can't be checked any way.